src/sidebar/store/modules/annotations.js

```diff
@@ -12,6 +12,9 @@ import * as util from '../util';
 /**
  * Return a copy of `current` with all matching annotations in `annotations`
  * removed.
+ *
+ * Annotations in `annotations` may be complete annotations or "stubs" with only
+ * the `id` field set.
  */
 function excludeAnnotations(current, annotations) {
```

Comment:
Given that this function's only caller is `removeAnnotations`, I think it could be further refactored for clarity or even integrated into `removeAnnotations`. That's a separate task, though!